packages/core-react/src/components/Type/type.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Type from '.';
+
+const SAMPLE_TEXT = 'All their equipment and instruments are alive.';
+
+describe('Type', () => {
+  test('it renders type with ray body class', () => {
+    const wrapper = mount(<Type>{SAMPLE_TEXT}</Type>);
+
+    expect(wrapper.find(Type).length).toBe(1);
```

Comment:
should you check that it has the ray body class since you are specifying that in `it renders type with ray body class`